src/directives/formly-form.js

```diff
@@ -369,6 +369,8 @@ function formlyForm(formlyUsability, formlyWarn, $parse, formlyConfig, $interpol
         options: field,
         index,
         formState: $scope.options.formState,
+        originalModel: '=?',
```

Comment:
I'm afraid this is incorrect. These values wont make sense in the context of a formly expression... @formly-js/angular-formly-collaborators, could someone provide more direction please?